### PR TITLE
Add "android" support (fix for bug #75)

### PR DESCRIPTION
**Pull request by oldosfan**
I must say, I was a little stupid, and somehow managed to let two commits get away without noticing

### CODE REVIEW

File: src/com/pty4j/unix/PtyHelpers.java [hunk 1]
```diff
@@ -208,14 +208,14 @@ else if (Platform.isFreeBSD()) {
...
-    else if (Platform.isLinux()) {
+    else if (Platform.isLinux() || Platform.isAndroid()) {
       myOsFacade = new com.pty4j.unix.linux.OSFacadeImpl();
     }
     else if (Platform.isWindows()) {
       throw new IllegalArgumentException("WinPtyProcess should be used on Windows");
     }
     else {
-      throw new RuntimeException("Pty4J has no support for OS " + System.getProperty("os.name"));
+      throw new RuntimeException("Pty4J has no support for OS " + System.getProperty("os.name") == "Linux" ? "This is abnormal" : System.getProperty("os.name"));
```
Comment (segrey) [inline src/com/pty4j/unix/PtyHelpers.java]: I suspect that `System.getProperty("os.name")` might return `Linux` in future even when `!Platform.isLinux() && !Platform.isAndroid() == true`. So, probably, it'd be better to keep the original error message (include `System.getProperty("os.name")` into the error message always).

**Comment by segrey** [comment]
Please squash commits.

File: src/com/pty4j/unix/PtyHelpers.java (hunk 1 above)
Comment (oldosfan) [reply src/com/pty4j/unix/PtyHelpers.java]: That's OK

**Comment by segrey** [comment]
@oldosfan Merged manually by https://github.com/JetBrains/pty4j/commit/6e039428be27bf24871358226751e293f3bce3aa.

**Comment by 10cl** [comment]
so, tell me, How to use it in Android?! help me~

**Comment by oldosfan** [comment]
Don't expect to be spoon fed. if you can't add a single dependency in a 
gradle buildscript you shouldn't be programming at all.

**Pull request closed without merging**